mostly patch level, except for opener and supports-hyperlinks.
can be reduced/dropped once those are updated.

Index: cargo/Cargo.toml
===================================================================
--- cargo.orig/Cargo.toml
+++ cargo/Cargo.toml
@@ -97,12 +97,12 @@ version = "0.4.67"
 version = "0.2.23"
 
 [dependencies.flate2]
-version = "1.0.30"
+version = "1.0.27"
 features = ["zlib"]
 default-features = false
 
 [dependencies.git2]
-version = "0.18.3"
+version = "0.18.2"
 
 [dependencies.git2-curl]
 version = "0.19.0"
@@ -128,10 +128,10 @@ version = "0.4.3"
 version = "0.12.1"
 
 [dependencies.home]
-version = "0.5.9"
+version = "0.5.5"
 
 [dependencies.http-auth]
-version = "0.1.9"
+version = "0.1.8"
 default-features = false
 
 [dependencies.humantime]
@@ -159,17 +159,17 @@ version = "1.3.0"
 version = "0.16.2"
 
 [dependencies.memchr]
-version = "2.7.2"
+version = "2.7.1"
 
 [dependencies.opener]
-version = "0.7.0"
+version = "0.6.0"
 
 [dependencies.os_info]
-version = "3.8.2"
+version = "3.7"
 default-features = false
 
 [dependencies.pasetors]
-version = "0.6.8"
+version = "0.6.7"
 features = [
     "v3",
     "paserk",
@@ -187,7 +187,7 @@ version = "0.8.5"
 version = "1.10.4"
 
 [dependencies.rusqlite]
-version = "0.31.0"
+version = "0.29.0"
 features = ["bundled"]
 
 [dependencies.rustfix]
@@ -197,7 +197,7 @@ version = "0.8.2"
 version = "1.0.6"
 
 [dependencies.semver]
-version = "1.0.22"
+version = "1.0.21"
 features = ["serde"]
 
 [dependencies.serde]
@@ -205,10 +205,10 @@ version = "1.0.199"
 features = ["derive"]
 
 [dependencies.serde-untagged]
-version = "0.1.5"
+version = "0.1.1"
 
 [dependencies.serde_ignored]
-version = "0.1.10"
+version = "0.1.2"
 
 [dependencies.serde_json]
 version = "1.0.116"
@@ -221,7 +221,7 @@ version = "0.10.6"
 version = "0.1.5"
 
 [dependencies.supports-hyperlinks]
-version = "3.0.0"
+version = "2.0.0"
 
 [dependencies.supports-unicode]
 version = "3.0.0"
@@ -234,7 +234,7 @@ default-features = false
 version = "3.10.1"
 
 [dependencies.time]
-version = "0.3.36"
+version = "0.3.31"
 features = [
     "parsing",
     "formatting",
@@ -256,7 +256,7 @@ version = "0.3.18"
 features = ["env-filter"]
 
 [dependencies.unicase]
-version = "2.7.0"
+version = "2.6.0"
 
 [dependencies.unicode-width]
 version = "0.1.12"
@@ -290,7 +290,7 @@ features = [
 ]
 
 [build-dependencies.flate2]
-version = "1.0.30"
+version = "1.0.27"
 features = ["zlib"]
 default-features = false
 
